src/homework/seventh/FirstTaskClass.java

```diff
@@ -0,0 +1,45 @@
+package homework.seventh;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class FirstTaskClass {
+
+    public static void main(String[] args) {
+        List<Integer> list = generateArray();
+        divideArray(list);
+    }
+
+    public static List<Integer> generateArray() {
+        Random random = new Random();
+        int randomLength = random.nextInt(1, 50);
+        List<Integer> array = new ArrayList<>(randomLength);
+
+        for (int i = 0; i < randomLength; i++) {
+            array.add(random.nextInt(0, 1000));
+        }
+        System.out.println(array);
+        return array;
+    }
+
+    public static List<Integer> divideArray(List<Integer> array) {
```

Comment:
According to the task array (int[]arr) should be as parameter

Reply:
public static void divideArrayElements(int[] array, int divisor) {
int[] dividedArray = new int[array.length];

    try {
        for (int i = 0; i < array.length; i++) {
            dividedArray[i] = array[i] / divisor;
        }
        System.out.println("Divided array elements:");
        for (int num : dividedArray) {
            System.out.println(num);
        }
    } catch (ArithmeticException e) {
        System.out.println("Error: Division by zero is not allowed.");
    } finally {
        System.out.println("Finally block executed.");
    }
}